packages/sanity/src/structure/panes/document/documentPanel/header/perspective/DocumentPerspectiveMenu.tsx

```diff
@@ -65,16 +43,16 @@ export function DocumentPerspectiveMenu(props: {documentId: string}): JSX.Elemen
           <BundleBadge hue={hue} title={title} icon={icon} padding={2} />
         </BadgeButton>
       )}
+
       {/** TODO IS THIS STILL NEEDED? VS THE PICKER IN STUDIO NAVBAR? */}
-      {documentVersions.length > 0 && (
-        <Box flex="none">
-          <BundleMenu
-            button={<Button icon={ChevronDownIcon} mode="bleed" padding={2} space={2} />}
-            bundles={documentVersions}
-            loading={!documentVersions}
-          />
-        </Box>
-      )}
+
+      <Box flex="none">
+        <BundleMenu
+          button={<Button icon={ChevronDownIcon} mode="bleed" padding={2} space={2} />}
+          bundles={documentVersions as BundleDocument[]}
```

Comment:
`BundleMenu` seems to accepts `BundleDocument[] | null`, and I think that's also the type of `documentVersions`, so might not need to do the type assertion here

Reply:
You're right, might have been because of something I did before messing around with the types, removed it :)